package.json

```diff
@@ -13,19 +13,19 @@
     "grenache"
   ],
   "dependencies": {
-    "async": "^2.6.0",
+    "async": "2.6.1",
     "bfx-facs-base": "git+https://github.com/bitfinexcom/bfx-facs-base.git",
-    "grenache-nodejs-http": "^0.7.6",
-    "lodash": "^4.17.10"
+    "grenache-nodejs-http": "1.0.1",
+    "lodash": "4.17.21"
```

Comment:
since we're here can we replace lodash with our lib?

Reply:
/cry